### ISSUE TITLE
Scales: classification for quantitative values

### ISSUE THREAD

**Issue by luucvanderzee** [spatialnetworkslab/vue-gg#129]
Now quantitative values are always scaled continuously. It is possible to do a binning data transformation, but this doesn't work well when you just want to keep the original data and not perform summaries. 

Maybe a syntax like

```
:fill="{ val: row.value, scale: { domain: 'value' }, classification: { <current binning transformation options> }"
```

Thoughts?

**Comment by atepoorthuis** [comment]
Yes, I think this syntax makes sense. It also means we need to add another prop to the scaling object. For many classification methods we don’t only need the `domain` but also the actual `values`.

**Comment by luucvanderzee** [comment]
@atepoorthuis true, but that is no issue. we already have access to those through the `DataInterface`, and we are already using all values for scales based on binned columns

**Comment by atepoorthuis** [comment]
I was thinking in the case of supplying a `domain` 'by hand', say:
```
scale: { domain: [0, 1000] , classification: { <binning transformation options> }
```
Then it isn't clear which values to to use. So if we want to keep the option open to define scales without direct reference to a variable in a dataset, we would need to add another prop, no?

**Comment by luucvanderzee** [comment]
hmm yeah... maybe we should then change the syntax to

`:fill="{ val: row.value, classification: { column: 'value', type: 'blues' binning: { <current binning transformation options> } }`

so that you can choose only `classification` OR `scale`. The word 'domain' is also not interchangeable with 'column' so maybe good to be explicit about this
